opentelemetry-zipkin/Cargo.toml

```diff
@@ -31,7 +31,7 @@ once_cell = { workspace = true }
 opentelemetry = { version = "0.23", path = "../opentelemetry" }
 opentelemetry_sdk = { version = "0.23", path = "../opentelemetry-sdk", features = ["trace"] }
 opentelemetry-http = { version = "0.12", path = "../opentelemetry-http" }
-opentelemetry-semantic-conventions = { version = "0.15", path = "../opentelemetry-semantic-conventions" }
+opentelemetry-semantic-conventions = { version = "0.16", path = "../opentelemetry-semantic-conventions" }
```

Comment:
Keep it "0.15" as of now. This version here would be bumped before publish.

Reply:
Unrelated to this PR - Zipkin exporter (just like other crates in this repo) shouldn't be using variables from semconv crate till it is stable. Will raise a separate PR for the cleanup.